internal/support/include/beforefirstline.bi

```diff
@@ -0,0 +1,218 @@
+'==========================================================================
+' This file is automatically included in each compiled program right before
+' the first program line, so to say it becomes line zero of each program.
+'----------
+' The purpose of this file is to implement common stuff into the language,
+' such as constants or type definitions. For this purpose this file allows
+' names beginning with an underscroe, which is usually not possible. However,
+' some rules have to be followed in order to avoid conflicts with user code.
+'----------
+' IMPORTANT
+'   The use of this file is exclusively reserved to QB64-PE itself,
+'   do NOT add your own personal stuff here !!
+'----------
+' RULES
+'   1.) Everything must be prepared to work with OPTION _EXPLICIT in effect,
+'       that is because this file is included in every tiny program and we
+'       never know if the user has OPTION _EXPLICIT set or not.
+'   2.) All used CONST, SUB/FUNCTION, TYPE and variable names must begin
+'       with an underscore, that is because those may conflict with CONST
+'       names. E.g. write a simple xyz = 123 here and it will throw an error
+'       as soon as the user defines a CONST xyz = ... in the global scope.
+'   3.) Exceptions from rule #2 are $LET variables, UDT elements and labels
+'       for GOTO/GOSUB etc., just choose unique names for it. However, if
+'       in doubt, a leading underscore can be used here too.
+' It's strongly recommended to edit this file in the QB64-PE editor only.
+' The IDE got adapted to automatically enforce the above rules whenever it
+' detects this file is loaded into the editor. I.e. it automatically enables
+' OPTION _EXPLICIT and it will throw "Invalid name" or "Invalid variable"
+' errors when missing a leading underscore.
+'==========================================================================
+
+$INCLUDEONCE
+
+'Boolean values
+CONST _TRUE = -1, _FALSE = 0
+'Relations (e.g. SGN and _STRCMP, _STRICMP)
+CONST _LESS = -1, _EQUAL = 0, _GREATER = 1
+'Special values
+CONST _NULL = 0
+CONST _STRING_EMPTY = ""
+CONST _STRING_NULL = CHR$(_NULL)
```

Comment:
Maybe `_CHR_NUL` instead? We could provide a `_CHR_QUOTE` and some others as well. Though I guess the user could just use `CHR$` with the `_ASC*` constants (and provide a `_ASC_NUL`).